flags/14.0/vtadmin.txt

```diff
@@ -0,0 +1,47 @@
+Usage:
+  vtadmin [flags]
+
+Flags:
+      --addr string                             address to serve on (default ":15000")
+      --alsologtostderr                         log to standard error as well as files
+      --cache-refresh-key string                instructs a request to ignore any cached data (if applicable) and refresh the cache;usable as an HTTP header named 'X-<key>' and as a gRPC metadata key '<key>'
+                                                Note: any whitespace characters are replaced with hyphens. (default "vt-cache-refresh")
+      --cluster cluster.ClustersFlag            per-cluster configuration. any values here take precedence over those in -cluster-defaults or -cluster-config (default [])
+      --cluster-config cluster.FileConfig       path to a yaml cluster configuration. see clusters.example.yaml (default {defaults: *cluster.Config:{ID: Name: DiscoveryImpl: DiscoveryFlagsByImpl:map[] TabletFQDNTmplStr: VtSQLFlags:map[] VtctldFlags:map[] BackupReadPoolConfig:<nil> SchemaReadPoolConfig:<nil> TopoRWPoolConfig:<nil> TopoReadPoolConfig:<nil> WorkflowReadPoolConfig:<nil> EmergencyFailoverPoolConfig:<nil> FailoverPoolConfig:<nil> SchemaCacheConfig:<nil> vtctldConfigOpts:[] vtsqlConfigOpts:[]}, clusters: []})
+      --cluster-defaults cluster.Config         default options for all clusters (default *cluster.Config:{ID: Name: DiscoveryImpl: DiscoveryFlagsByImpl:map[] TabletFQDNTmplStr: VtSQLFlags:map[] VtctldFlags:map[] BackupReadPoolConfig:<nil> SchemaReadPoolConfig:<nil> TopoRWPoolConfig:<nil> TopoReadPoolConfig:<nil> WorkflowReadPoolConfig:<nil> EmergencyFailoverPoolConfig:<nil> FailoverPoolConfig:<nil> SchemaCacheConfig:<nil> vtctldConfigOpts:[] vtsqlConfigOpts:[]})
+      --datadog-agent-host string               host to send spans to. if empty, no tracing will be done
+      --datadog-agent-port string               port to send spans to. if empty, no tracing will be done
+      --emit_stats                              If set, emit stats to push-based monitoring and stats backends
+      --enable-dynamic-clusters                 whether to enable dynamic clusters that are set by request header cookies or gRPC metadata
+      --grpc-allow-reflection grpc_cli          whether to register the gRPC server for reflection; this is required to use tools like grpc_cli
```

Comment:
shoot, i know what caused this weird output (it's suggesting that the _type_ of this is `grpc_cli` instead of bool)

Reply:
#11464

Reply:
merged (thanks for that, deepthi!)